### ISSUE TITLE
verbb\navigation\nodetypes\PassiveType is not a valid element

### ISSUE THREAD

**Issue by MattWilcox** [verbb/navigation#257]
**Description**

A client's been adding data to their staging website, and we are now unable to get into the Navigation for it in the CMS. 

http://project.nitro/admin/navigation/navs/build/1

spits out...

```
Invalid Argument – yii\base\InvalidArgumentException
verbb\navigation\nodetypes\PassiveType is not a valid element.
```

**Additional info**
- Plugin version: 1.4.15
- Craft version: 3.6.13
- Multi-site: No

**Comment by engram-design** [comment]
Are you able to provide a full stack trace of this error? As in a screenshot of the error with devMode on?

Or - have a look for this error in your storage/logs files. I'm interested in where this is being fired from, as whatever is triggering this is trying to use a passive node as an element, which it isn't.

**Comment by engram-design** [comment]
Actually, should be fixed in [1.4.16](https://github.com/verbb/navigation/releases/tag/1.4.16).

This seems to be due to an element node getting changed to a passive node (or other non-element node), but still retaining its element information `elementId` column in the database. This shouldn't be happening, and there's checks in place to whenever you save a node to clear this info out, but there's a potential that this change was done before the latest Navigation version where this was patched.

**Comment by MattWilcox** [comment]
Can confirm this is fixed. Thanks!
